[2023-09-23T17:05:54.611+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: load_data.load_to_fact_table manual__2023-09-23T17:05:53.429893+00:00 [queued]>
[2023-09-23T17:05:54.617+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: load_data.load_to_fact_table manual__2023-09-23T17:05:53.429893+00:00 [queued]>
[2023-09-23T17:05:54.617+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-23T17:05:54.627+0000] {taskinstance.py:1327} INFO - Executing <Task(_PythonDecoratedOperator): load_to_fact_table> on 2023-09-23 17:05:53.429893+00:00
[2023-09-23T17:05:54.632+0000] {standard_task_runner.py:57} INFO - Started process 3713 to run task
[2023-09-23T17:05:54.634+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'load_data', 'load_to_fact_table', 'manual__2023-09-23T17:05:53.429893+00:00', '--job-id', '5456', '--raw', '--subdir', 'DAGS_FOLDER/load_data.py', '--cfg-path', '/tmp/tmp2d8i75nw']
[2023-09-23T17:05:54.638+0000] {standard_task_runner.py:85} INFO - Job 5456: Subtask load_to_fact_table
[2023-09-23T17:05:54.670+0000] {task_command.py:410} INFO - Running <TaskInstance: load_data.load_to_fact_table manual__2023-09-23T17:05:53.429893+00:00 [running]> on host 833e75d2e24d
[2023-09-23T17:05:54.731+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='load_data' AIRFLOW_CTX_TASK_ID='load_to_fact_table' AIRFLOW_CTX_EXECUTION_DATE='2023-09-23T17:05:53.429893+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-23T17:05:53.429893+00:00'
[2023-09-23T17:05:56.515+0000] {logging_mixin.py:149} INFO - Connection Succesful
[2023-09-23T17:05:58.499+0000] {logging_mixin.py:149} INFO - Loaded 1 rows to BigQuery
[2023-09-23T17:06:02.520+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-23T17:06:02.529+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=load_data, task_id=load_to_fact_table, execution_date=20230923T170553, start_date=20230923T170554, end_date=20230923T170602
[2023-09-23T17:06:02.547+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-23T17:06:02.561+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
